select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -465,4 +465,12 @@ public void onComponentTagBody(final MarkupStream markupStream, final ComponentT
         }
 		replaceComponentTagBody(markupStream, openTag, buffer);
 	}
+	
+	/**
+	 * Empty input is acceptable
+	 */
+	@Override
+	public boolean isInputNullable() {
+		return true;
+	}
```

Comment:
@martin-g , if input is empty I would like to expect empty list:) That was a problem.
This fix has been tested: if you remove previously entered tags: at the end instead of showing everything it's showing empty. I think it's pretty natural: other wise it leads to bugs. Let me show real example:
1) Go to this page: http://demo.orienteer.org/class/MyClass  (login as admin/admin)
2) Click Edit
3) Try to remove ALL super classes.